yt_dlp/extractor/redgifs.py

```diff
@@ -98,10 +103,12 @@ class RedGifsIE(RedGifsBaseInfoExtractor):
             'upload_date': '20211107',
             'uploader': 'ignored52',
             'duration': 16,
-            'view_count': int,
+            'view_count': None,
             'like_count': int,
             'categories': list,
             'age_limit': 18,
+            'ie_key': 'RedGifs',
+            'tags': list,
```

Comment:
```suggestion
            'tags': list,
```

Reply:
The new key returned is actually in the data. The test fails if I don't add this because of the extra key.